datacenter/ucp/2.0/guides/user-management/permission-levels.md

```diff
@@ -43,8 +43,7 @@ users can see the containers they deploy in the cluster.
 
 ## Team permission levels
 
-Teams allow you to define fine-grain permissions to services, containers, and
-networks that have the label `com.docker.ucp.access.label` applied to them.
+Teams and Labels give the administrator fine-grained control over permissions. Each team can have multiple labels. Each label has a key of `com.docker.ucp.access.label`. The label is then applied to the containers, services, networks, secrets and volumes. 
```

Comment:
Can we also make it explicit what resources don't have RBAC? Is it just images and nodes @alexmavr?